lib/MHA/Config.pm

```diff
@@ -263,6 +263,12 @@ sub parse_server {
   $value{skip_init_ssh_check} = 0
     if ( !defined( $value{skip_init_ssh_check} ) );
 
+  $value{skip_version_check} = $param_arg->{skip_version_check};
+  $value{skip_version_check} = $default->{skip_version_check}
+    if ( !defined( $value{skip_version_check} ) );
```

Comment:
I believe so: the top statement will load the config from the individual server's config. The second one will use the default from `[server default]` if it exists, and the third statement (not pictured) will put in a 0 in case it's not defined anywhere.

This lets you override the version check on a per server or default-for-all-servers basis.